src/chi-vue/src/components/data-table-toolbar/DataTableToolbar.tsx

```diff
@@ -53,7 +55,7 @@ export default class DataTableToolbar extends Vue {
   render() {
     return (
       <div class={`${DATA_TABLE_CLASSES.TOOLBAR} ${PORTAL_CLASS}`}>
-        <div class={`${DATA_TABLE_CLASSES.TOOLBAR}__header`}>
+        <div class={this.ribbon ? `${DATA_TABLE_CLASSES.TOOLBAR}__ribbon` : `${DATA_TABLE_CLASSES.TOOLBAR}__header`}>
```

Comment:
@evansito Can you elaborate on the logic here? Why do we need this condition?

Reply:
Yes, to use the ribbon class when applicable but i can see your point!